dd-sdk-android-androidx/src/main/kotlin/com/datadog/android/androidx/tracking/JetpackExtraAttributesProvider.kt

```diff
@@ -0,0 +1,42 @@
+package com.datadog.android.androidx.tracking
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import com.datadog.android.rum.tracking.ViewAttributesProvider
+
+/**
+ * Provides extra attributes for the touch target View.
+ * Those special attributes are specifically related with Jetpack components.
+ * Examples:
+ * <ul>
+ *     <li> If the parent of the target view is a RecyclerView it will add the position
+ *     of the target inside the adapter as a special attribute </li>
+ * </ul>
+ * @see [TAG_TARGET_ADAPTER_POSITION]
+ */
+object JetpackExtraAttributesProvider : ViewAttributesProvider {
```

Comment:
I'd rename it as `JetpackViewAttributesProvider` for consistency with `ViewAttributesProvider`